src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import duke.*;
```

Comment:
please import explicitly instead of using *. Check your other imports as well with the same issue